### PR TITLE
Makes sniper rifle descriptions less shit

### PR DESCRIPTION
**Pull request by ghost**
these shitty sniper rifle descriptions are so fucking edgy i could use them as a substitute for the esword

:cl: Batman
tweak: The sniper rifle descriptions are less edgy.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -352,7 +352,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/sniper_rifle
 	name = "sniper rifle"
-	desc = "The kind of gun that will leave you crying for mummy before you even realise your leg's missing"
```
Comment (PKPenguin321) [inline code/modules/projectiles/guns/projectile/automatic.dm]: who even writes this trash

Comment (RemieRichards) [inline code/modules/projectiles/guns/projectile/automatic.dm]: I did, nobody gave a shit (least of all me descriptions are always shit), you missed your chance to complain tbh.

Comment (RemieRichards) [inline code/modules/projectiles/guns/projectile/automatic.dm]: It has to deal 999 damage, to continue the trend that I suck at balance.
Remember the original acid act where all your shit turned to goo? that's me!
Fire spreading on touch? Meeee~

Also it'd be a player ref and I already have a few of those :P

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 2]
```diff
@@ -352,7 +352,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/sniper_rifle
 	name = "sniper rifle"
-	desc = "The kind of gun that will leave you crying for mummy before you even realise your leg's missing"
+	desc = "A long ranged weapon that does significant damage. No, you can't quickscope."
```
Comment (kevinz000) [inline code/modules/projectiles/guns/projectile/automatic.dm]: ynot

**Review by Iamgoofball — APPROVED** [review]
quickscope

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/projectiles/guns/projectile/automatic.dm]: remie you've committed a mortal sin to our codebase you must repent

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 2 above)
Comment (optimumtact) [inline code/modules/projectiles/guns/projectile/automatic.dm]: it's always in scoped mode

**Comment by Militaires** [comment]
spelling mistake with suppressed on syndie rifle desc

**Review by silicons — APPROVED** [review]

**Pull request merged**